//
// Academic License - for use in teaching, academic research, and meeting
// course requirements at degree granting institutions only.  Not for
// government, commercial, or other organizational use.
//
// abcef_O_VP.h
//
// Code generation for function 'abcef_O_VP'
//

#ifndef ABCEF_O_VP_H
#define ABCEF_O_VP_H

// Include files
#include "rtwtypes.h"
#include <cstddef>
#include <cstdlib>

// Function Declarations
void abcef_O_VP(double P_init, double V_init, double A_init, double P_wayp,
                double V_wayp, double A_max, double A_min, double J_max,
                double J_min, creal_T t[14]);

#endif
// End of code generation (abcef_O_VP.h)
